### ISSUE TITLE
When listing members of an environment which has members that are delete , invalidIdentity (400) error is thrown.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#2022]
Server version - V0.37.0-rc2

Local auth enabled setup

As admin, created user - test1
Made "test1" member of default project.
Deactivated and deleted "test1".
"test1" does not have access to server as expected.

As admin , go to "Manage Environment" and click on "Default" project ,

User is directed to following error message:

"Error
invalidIdentity (400)
Rancher Account9nonexistent"

API call made from UI:
Request URL:http://<server ip>:8080/v1/projects/1a5/projectmembers
Request Method:GET
Status Code:400 Bad Request"

"Error
invalidIdentity (400)
Rancher Account9nonexistent"

```
mysql> select * from account;
+----+-----------------------+------------+--------------------------------------+-------------+----------+---------------------+---------------------+---------------------+------------------------------------------------------------+-------------+------------------+
| id | name                  | kind       | uuid                                 | description | state    | created             | removed             | remove_time         | data                                                       | external_id | external_id_type |
+----+-----------------------+------------+--------------------------------------+-------------+----------+---------------------+---------------------+---------------------+------------------------------------------------------------+-------------+------------------+
|  1 | hariharan             | admin      | admin                                | NULL        | active   | NULL                | NULL                | NULL                | {"fields":{"hasLoggedIn":true}}                            | 1           | rancher_id       |
|  2 | system                | system     | system                               | NULL        | inactive | NULL                | NULL                | NULL                | NULL                                                       | NULL        | NULL             |
|  3 | superadmin            | superadmin | superadmin                           | NULL        | inactive | NULL                | NULL                | NULL                | NULL                                                       | NULL        | NULL             |
|  4 | token                 | token      | token                                | NULL        | active   | 2015-09-10 23:47:30 | NULL                | NULL                | {}                                                         | NULL        | NULL             |
|  5 | Default               | project    | adminProject                         | NULL        | active   | 2015-09-10 23:47:30 | NULL                | NULL                | {"fields":{"defaultNetworkId":5,"capabilities":["owner"]}} | NULL        | NULL             |
|  6 | NULL                  | agent      | agentAccount1                        | NULL        | active   | 2015-09-10 23:49:26 | NULL                | NULL                | {}                                                         | NULL        | NULL             |
|  7 | NULL                  | agent      | agentAccount2                        | NULL        | active   | 2015-09-10 23:49:35 | NULL                | NULL                | {}                                                         | NULL        | NULL             |
|  8 | machineServiceAccount | service    | machineServiceAccount                | NULL        | active   | 2015-09-10 23:49:50 | NULL                | NULL                | {}                                                         | NULL        | NULL             |
|  9 | test1                 | user       | 20c36720-d1f8-4901-9b02-0e477ab9f92a | test1       | purged   | 2015-09-10 23:50:55 | 2015-09-10 23:52:06 | 2015-09-10 23:53:06 | {"fields":{"hasLoggedIn":true}}                            | 9           | rancher_id       |
+----+-----------------------+------------+--------------------------------------+-------------+----------+---------------------+---------------------+---------------------+------------------------------------------------------------+-------------+------------------+
9 rows in set (0.00 sec)

mysql> select * from project_members;
ERROR 1146 (42S02): Table 'cattle.project_members' doesn't exist
mysql> select * from project_member; 
+----+-----------+------------+---------------+--------------------------------------+-------------+--------+---------------------+---------------------+---------------------+------+-------------+------------+------------------+--------+
| id | name      | account_id | kind          | uuid                                 | description | state  | created             | removed             | remove_time         | data | external_id | project_id | external_id_type | role   |
+----+-----------+------------+---------------+--------------------------------------+-------------+--------+---------------------+---------------------+---------------------+------+-------------+------------+------------------+--------+
|  1 | NULL      |          5 | projectMember | adminMember                          | NULL        | purged | 2015-09-10 23:47:30 | 2015-09-10 23:50:30 | 2015-09-10 23:51:30 | {}   | 1           |          5 | rancher_id       | owner  |
|  2 | hariharan |          4 | projectMember | 8d796855-713a-44b8-bed4-0e5319b6049d | NULL        | active | 2015-09-10 23:50:30 | NULL                | NULL                | {}   | 1           |          5 | rancher_id       | owner  |
|  3 | test1     |          5 | projectMember | 1cb9e934-5be9-47d1-978c-dcc435de1800 | NULL        | active | 2015-09-10 23:51:08 | NULL                | NULL                | {}   | 9           |          5 | rancher_id       | member |
+----+-----------+------------+---------------+--------------------------------------+-------------+--------+---------------------+---------------------+---------------------+------+-------------+------------+------------------+--------+
3 rows in set (0.00 sec)

```

**Comment by hibooboo2** [comment]
Addressed with rancher/cattle#863

**Comment by sangeethah** [comment]
Tested with Server version - V0.37.0-rc3

When listing members of an environment , members that are deleted are listed with (ID not found) appended to the name field.
